.drone.yml

```diff
@@ -381,6 +381,23 @@ steps:
     event:
       - push
 
+- name: upload_kernel_initrd_releases
+  image: plugins/github-release
+  settings:
+    prerelease: true
```

Comment:
As discussed in DM, we also need the `draft` setting set to `true` here.
Usually, we tag first and publish the corresponding release if everything goes well.

Reply:
thanks for the spot. this is now fixed @bk201